src/electionguard/serializable.py

```diff
@@ -1,21 +1,27 @@
-from os import path
 from dataclasses import dataclass
+from datetime import datetime
+from os import path
+from typing import cast, TypeVar, Generic
+
 from jsons import (
-    dump,
     dumps,
+    NoneType,
     loads,
     JsonsError,
     set_deserializer,
     set_serializer,
     set_validator,
+    default_nonetype_deserializer,
 )
-from typing import cast, TypeVar, Generic
 
 T = TypeVar("T")
 
 JSON_FILE_EXTENSION: str = ".json"
 WRITE: str = "w"
+READ: str = "r"
 JSON_PARSE_ERROR = '{"error": "Object could not be parsed due to json issue"}'
+# TODO Issue #??: Jsons library incorrectly dumps class method
+FROM_JSON_FILE = '"from_json_file": {}, '
```

Comment:
This is handling a bug with `jsons` that an issue still needs to be made for. I'll update PR accordingly. 

Essentially every output file has this classmethod appended.